19.12

```diff
@@ -0,0 +1,106 @@
+#%Module1.0
+# The first line of this file tells Modules that this is a module file.
+# DO NOT ALTER IT!
+#
+# METADATA AND DOCUMENTATION SECTION
+#
+# This function is part of the Modules help system.  You can modify
+# the second line if needed, but most products should
+# leave this alone.
+#
+proc ModulesHelp { } {
+    global product version
+    puts stderr "This module adds $product/$version to your environment."
+}
+#
+# These variables are used below.  The product variable should be set to
+# the name of the product and never changed.  The version variable will
+# be set at install time, so it should be left alone.  The conflict line
+# prevents multiple versions from being loaded simultaneously.  Do not
+# change it.
+#
+set product desimodules
+set version 19.12
+conflict $product
+set desiconda_version 20190804-1.3.0-spec
+#
+# The line below is another part of the Modules help system.  You can
+# modify the part in quotes if you really need to, but most products should
+# leave this alone.
+#
+module-whatis "Sets up $product/$version in your environment."
+#
+# DEPENDENCIES SECTION
+#
+# If your product requires other software to function, that should be declared
+# here.  There are two types of dependencies: mandatory and optional.
+# A mandatory dependency is a module load command followed by a prereq
+# command.  An optional dependency is not followed by a prereq statement.
+#
+# NO DEPENDENCIES
+#
+# ENVIRONMENT SECTION
+#
+# The PRODUCT_ROOT and PRODUCT_DIR variables are used to set other
+# environment variables but are not exported to the actual environment.
+# If you are not working at NERSC, but still want to use Modules, you
+# will need to set the DESI_PRODUCT_ROOT environment variable
+#
+if {[info exists env(DESI_PRODUCT_ROOT)]} {
+    set code_root $env(DESI_PRODUCT_ROOT)/code
+} else {
+    set code_root /global/common/software/desi/$env(NERSC_HOST)/desiconda/$desiconda_version/code
+}
+# set PRODUCT_DIR $code_root/$product/$version
+set PRODUCT_DIR /global/common/software/desi/$env(NERSC_HOST)/desiconda/startup/modulefiles/$product
+#
+# This line creates an environment variable pointing to the install
+# directory of your product.
+#
+setenv [string toupper $product] $PRODUCT_DIR
+setenv [string toupper $product]_VERSION $version
+#
+# The lines below set various other environment variables.  They assume the
+# template product layout.  These will be set or commented as needed by the
+# desiInstall script.
+#
+# prepend-path PATH $PRODUCT_DIR/bin
+# prepend-path PYTHONPATH $PRODUCT_DIR/lib/python3.5/site-packages
+# prepend-path PYTHONPATH $PRODUCT_DIR/py
+# prepend-path LD_LIBRARY_PATH $PRODUCT_DIR/lib
+# prepend-path IDL_PATH +$PRODUCT_DIR/pro
```

Comment:
Let's go ahead and remove these lines since they don't apply to desimodules and refer to an outdated python3.5 anyway; we're now using python 3.6.6.  If there is a strong reason to keep them for maintenance consistency with other code modules, ok.